config.sample.php

```diff
@@ -475,6 +475,47 @@ public static function isOnPlayNowDomain()
 		}
 		return false;	
 	}
-}
 
+	// ---
```

Comment:
Are the various keys and secrets here something that we will have to note for the merge and deploy? e.g. they are something Kestas will need to privately pick some secure values for the production version of the config upon merge and deploy of a version of webdip after this change is merged in?

Reply:
Good point. Do you know if he manages that using ENV variables or some other ENV variables manager?

Reply:
No idea. Just a thing to note somewhere so that we don't forget when it comes time to do the next merge and release.